clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2705,6 +2705,30 @@ An error will be given if:
   }];
 }
 
+def AMDGPUNumWorkGroupsDocs : Documentation {
+  let Category = DocCatAMDGPUAttributes;
+  let Content = [{
+The number of work groups specifies the number of work groups when the kernel
```

Comment:
I think this needs more elaboration... "The number of work groups specifies the number of work groups" is a tautology.

Reply:
Updated. However, the phrase "number of work groups" itself appears to be clear enough.